src/main/java/uk/co/downthewire/jLTE/simulator/FadingData.java

```diff
@@ -47,19 +52,22 @@ public void readFading(int iteration) throws IOException {
         int modifiedIteration = iteration % 120;
```

Comment:
@thompsy FMI, do you mind explaining why you are taking modulo 120 here? What's the meaning of the number 120?